apps/wallet-mobile/src/Staking/PoolTransition/usePoolTransition.tsx

```diff
@@ -0,0 +1,426 @@
+        <LinearGradient
+          style={[StyleSheet.absoluteFill, {opacity: 1, borderRadius: 8}]}
+          start={{x: 0, y: 0}}
+          end={{x: 1, y: 0.5}}
+          colors={colors.backgroundGradientCard}
+        />
+
+        <View style={styles.row}>
+          <Text style={[styles.regular, styles.gray]}>{strings.newPool}</Text>
+
+          <View style={styles.tickerWrap}>
+            {poolTransition.suggestedPool.pic != null && (
+              <Image source={{uri: poolTransition.suggestedPool.pic}} style={styles.pic} />
+            )}
+
+            <Text
+              style={[styles.regular, styles.primary]}
+            >{`[${poolTransition.suggestedPool.ticker}] ${poolTransition.suggestedPool.name}`}</Text>
+          </View>
+        </View>
+
+        <View style={styles.row}>
+          <Text style={[styles.regular, styles.gray]}>{strings.estimatedRoa}</Text>
+
+          <Text style={[styles.bold, styles.max]}>{poolTransition.suggestedPool.roa} %</Text>
+        </View>
+
+        <View style={styles.row}>
+          <Text style={[styles.regular, styles.gray]}>{strings.fee}</Text>
+
+          <Text style={[styles.bold, styles.max]}>{poolTransition.suggestedPool.tax} %</Text>
+        </View>
+
+        <View>
+          <Text style={[styles.regular, styles.max]}>{strings.poolGeneratesRewards}</Text>
+        </View>
+      </View>
+
+      <Spacer height={24} />
+
+      <View style={styles.buttonContainer}>
+        <Button outline title={strings.skipNoRewards} textStyles={styles.outlineButton} onPress={handleOnSkip} />
+
+        <Spacer height={6} />
+
+        <Button shelleyTheme title={strings.updateKeepEarning} onPress={handleOnUpdate} textStyles={styles.button} />
+      </View>
+
+      <Spacer height={24} />
+    </View>
+  )
+}
+
+const useStyles = () => {
+  const {theme} = useTheme()
+  const modalHeight = 700
+
+  const styles = StyleSheet.create({
+    modal: {
+      backgroundColor: theme.color['bottom-sheet-background'],
+      alignItems: 'center',
+      flex: 1,
+      justifyContent: 'flex-end',
+    },
+    card: {
+      borderRadius: 8,
+      gap: 8,
+      ...theme.padding.l,
+      width: '100%',
+      overflow: 'hidden',
+    },
+    border: {
+      borderWidth: 1,
+      borderColor: theme.color.gray['300'],
+    },
+    redBorder: {
+      borderWidth: 1,
+      borderColor: theme.color.magenta['500'],
+    },
+    tickerWrap: {
+      alignItems: 'center',
+      flexDirection: 'row',
+      gap: 8,
+    },
+    pic: {
+      width: 24,
+      height: 24,
+      borderRadius: 100,
+    },
+    regular: {
+      ...theme.typography['body-1-l-regular'],
+    },
+    bold: {
+      ...theme.typography['body-1-l-medium'],
+    },
+    gray: {
+      color: theme.color.gray['600'],
+    },
+    max: {
+      color: theme.color.gray['max'],
+    },
+    red: {
+      color: theme.color.magenta['500'],
+    },
+    primary: {
+      color: theme.color.primary[600],
+    },
```

Comment:
into -> colors

Reply:
Not sure I understand this one. They're colors, but I want to use them as styles for text. If a move them to the colors object, I'd still need them as styles for the text.

Reply:
so why giving a text style a color name? should it be warningText, errorText?

Reply:
I mean, other than being longer and having to figure out a name for them, no reason.
The design has in a single portion of the screen 7 different combinations of color and boldness. So just trying to be simple with the names.
I can totally append `Text` to all of them.

Reply:
![image](https://github.com/Emurgo/yoroi/assets/1725956/49c2d05e-40a4-4616-bf79-56970e36dd0b)

Well they represent `text`, `textInsideCard`, `rowLabel`, `poolName`, `rowCurrentValue`, `rowSuggestedValue`, `warningText`, `warningTextTime`. I disagree on that being better, but I can change them all.

Reply:
these are the places where the colors are implemented, my question is more, is there a word that represents what it is visually trying to indicate? maybe it's too many places then I agree, let's keep it, but if it's possible to identify what is representing red -> warning, gray -> labels, but if it's spread no worries